FF1Lib/MapExchange/OwMapExchange.cs

```diff
@@ -9,6 +9,7 @@
 using System.Reflection;
 using System.ComponentModel;
 using RomUtilities;
+using FF1Lib.Procgen;
```

Comment:
can't comment below:

foreach (var df in data.DomainFixups) domains.SwapDomains(df.From, df.To);
foreach (var df in data.DomainUpdates) domains.Data[df.To] = originalDomains.Data[df.From];

Please add some null checks here.

Reply:
Sure thing.  I worked around it by assigning an empty array to DomainFixups but it would certainly be better to not crash there!